### PR TITLE
Reformatted project struct

### PR DESCRIPTION
**Pull request by clundin55**
Notes of things I would like everyone's opinion on.
I would like to rename yw_website to website and apps/website to apps/yesworkflow_web

### CODE REVIEW

File: yw_website/yw_website/apps/website/management/commands/create_test_database.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
     subgraph cluster_workflow_box_outer { label=""; color=black; penwidth=2
     subgraph cluster_workflow_box_inner { label=""; penwidth=0
     node[shape=box style=filled fillcolor="#CCFFCC" peripheries=1 fontname=Helvetica]
-    node[shape=box style=filled fillcolor="#CCFFCC" peripheries=2 fontname=Helvetica]
+    node[shape=box style=filled fillolor="#CCFFCC" peripheries=2 fontname=Helvetica]
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/management/commands/create_test_database.py]: typo, this could effect the generated test graphs

Comment (clundin55) [reply yw_website/yw_website/apps/website/management/commands/create_test_database.py]: Oops

**Review by aniehuser — APPROVED** [review]
Looks like everything still works, changes are signficant however, so we all need to clone a new copy of the repo after this is merged @clundin55 @mbrown20 @Brewer-Slack @bthuntington

**Comment by clundin55** [comment]
@aniehuser @mbrown20 @Brewer-Slack @bthuntington you can also type 

git reset --hard origin/master if you don't want to reclone.

**Pull request merged**